### ISSUE TITLE
websocket: close 1006 again

### ISSUE THREAD

**Issue by Lucipher** [Rhymen/go-whatsapp#191]
connection to WhatsApp servers failed: websocket: close 1006 (abnormal closure): unexpected EOFkeepAlive failed: connection timed

go wac.keepAlive(20000, 90000) -> go wac.keepAlive(20000, 60000)
not work.

**Comment by tarikhagustia** [comment]
i have same issue after running up to 6 hours

**Comment by Lucipher** [comment]
Yes, it need to has a schedule job to check the connection  status.

**Comment by bryanjack** [comment]
Same Issue

error caught in handler: connection to WhatsApp servers failed: websocket: close 1006 (abnormal closure): unexpected EOF

**Comment by fredjsf** [comment]
me too

**Comment by Lucipher** [comment]
try to reconnect when error
